python/.pre-commit-config.yaml

```diff
@@ -1,7 +1,6 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.6.0
-    files: ^python/
```

Comment:
I think we can exclude this restriction now since integrations have been or will be moved.